walletdatabase/migrations/sql/1712763613_add_collection_socials.up.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE IF NOT EXISTS collection_socials_cache (
```

Comment:
How do we clean this table and ensure we don't leave dangling data forever? Won't it need some foreign key to the collections table?